modules/rank-eval/src/test/java/org/elasticsearch/index/rankeval/RankEvalRequestTests.java

```diff
@@ -35,7 +35,15 @@
 import java.util.Map.Entry;
 import java.util.Set;
 
-@ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.SUITE)
+@ESIntegTestCase.ClusterScope(
+        scope = ESIntegTestCase.Scope.SUITE,
+        minNumDataNodes = 1,
+        numDataNodes = 2,
+        maxNumDataNodes = 3,
+        numClientNodes = 1,
+        transportClientRatio = 0.5,
+        supportsDedicatedMasters = true,
+        randomDynamicTemplates = true)
```

Comment:
I just tested w/o this annotation, things looked good. If I'm not entirely mistaken I introduced it before the code worked w/ the transportclient.